src/unix/bsd/freebsdlike/freebsd/x86_64/mod.rs

```diff
@@ -78,6 +80,16 @@ s_no_extra_traits! {
         pub xmm_reg: [[u8; 16]; 8],
         pub xmm_pad: [u8; 224],
     }
+
+    #[cfg_attr(feature = "extra_traits", derive(Debug))]
+    pub struct ucontext_t {
+        pub uc_sigmask: ::sigset_t,
+        pub uc_mcontext: ::mcontext_t,
```

Comment:
This causes importing failure:
```
error[E0412]: cannot find type `mcontext_t` in the crate root
  --> src/unix/bsd/freebsdlike/freebsd/x86_64/mod.rs:87:28
   |
87 |         pub uc_mcontext: ::mcontext_t,
   |                            ^^^^^^^^^^ did you mean `ucontext_t`?
```

Reply:
I guess these ancient rustcs were single pass and the `pub use self::align::*` at the bottom doesn't bring it into this scope?!

Screw it, moving `ucontext_t` into the align module too.